ios/atb.xcodeproj/project.pbxproj

```diff
@@ -319,22 +270,14 @@
 				TargetAttributes = {
 					00E356ED1AD99517003FC87E = {
 						CreatedOnToolsVersion = 6.2;
+						DevelopmentTeam = 7U3L7LAA9V;
```

Comment:
Denne må endres også

Reply:
Ja, så det selv. Men finner den ikke i xcode. Hvis du ser under denne så ligger devTeam id for NFK også.
Skal ta en titt til.

Reply:
Har du noen ide om hvor denne settes?

Reply:
Tror denne er for prosjekt og ikke target, så burde ikke ha noe å si, men har endret den til å bruke IOS_DEVELOPMENT_TEAM_ID. Den blir satt riktig når man kjører script, men den endres ikke i pbxproj filen.

Reply:
Jeg endte opp med bare å slette de manuelt, per nå klarer jeg ikke finne noen grunn til at de må være der